### PR TITLE
Do you speak PS - May 2017 edition

### PR DESCRIPTION
**Pull request by AlexEven**

### CODE REVIEW

File: news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+A lot of words have been translated since the beginning of the year :)
+Here's a quick overview of the evolution of the most active languages over the past few months - Thanks for your involvement!
+
+
+![Crowdin Monthly banner](/assets/images/2017/06/PrestaShop_translation_evolution_2017.gif)
+
+### Deactivating Swiss-German in PrestaShop
+
+It's been a while the German for Switzerland is avalaible for Switzerland merchants. However, this language does not bring any value to merchants at the moment: Indeed, it is similar to the German language, and hasn't been localized to answer the needs of the Swiss market. Confusing.
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md]: "It's been a while _that_ the German (...)" ?
Also needs to end with a br tag if you want a linefeed.

File: news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+Thus, from 1.7.2.O onwards, the Swiss localization package will load the German language only.
+
+The German, Switzerland translation project has also been removed from the Crowdin translation project. It has never really been maintained and it remained inactive for too long.
+
+If in the future, if merchants want to this language back, we'd be happy to re-open the language on Crowdin :)
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md]: Twice as many "if"s! :)

File: news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+* Swahili, Tanzania
+* Telugu
+
+Again, if you wish to contribute for one of these languages, please drop us a word at translation@prestashop.com, and this language will be available for translation again on Crowdin.
+
+### 1.7.2 on its way
+
+As per our release planning, we're preparing version 1.7.2.0.
+
+Before it's release, we will upload its strings to the Crowdin project:
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md]: "its"

Markdown requires an empty line before a list.

File: news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+* so that translators have the time to translate this version before it's release
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md]: 7d3795b0f340a3a65785550d6732ddb3

File: news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+* to check the content and see if any modifications are needed before the release
+
+There will be new content, mostly for Stock Management (back in 1.7.2.0), so your help will be required.
```
Comment (xBorderie) [inline news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md]: "There will be new content to translate soon, mostly for the Stock Management feature (which is making a come back in 1.7.2.0)" + link to [Guillaume's latest article](http://build.prestashop.com/news/stock-management-in-prestashop-1-7-part-2/)?

**Comment by AlexEven** [comment]
Thanks @xBorderie for the review :grin:

**Pull request merged**